content/platforms/kubernetes/pod-scheduling.md

```diff
@@ -0,0 +1,187 @@
+cluster.
+
+You can override this rule to restrict or widen the nodes on which Redis Enterprise
+cluster node pods will run. For example, you could restrict your cluster so that
+no Redis Enterprise node pod is scheduled on the same K8s node regardless of the
+cluster it belongs to by just deleting the `redis.io/cluster` label:
+
+```yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: rec
+spec:
+  size: 3
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+    - labelSelector:
+        matchLabels:
+          app: redis-enterprise
+          redis.io/role: node
+      topologyKey: kubernetes.io/hostname
+```
+
+or you could avoid placing Redis Enterprise nodes with other databases:
+
+```yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: rec
+spec:
+  size: 3
+  extraLabels:
+     local/role: database
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+    - labelSelector:
+        matchLabels:
+          local/role: database
+          app: redis-enterprise
+          redis.io/cluster: rec
+          redis.io/role: node
+      topologyKey: kubernetes.io/hostname
+```
+
+If other database's pods are deployed with the label `local/role: database`,
+the Redis Enterprise pods will also not be co-located on the same node.
+
+
+## Using rack awareness
+
+Redis Enterprise can be configured to be rack aware so that the availability
+is increased in the event of partitions or other rack (or region) related
+failures.
+
+Rack awareness is a single property in the Redis Enterprise cluster CRD named
+`rackAwarenessNodeLabel` whose value is the rack label applied to the node. This
+label is often `failure-domain.beta.kubernetes.io/zone` as documented in
+['Running in multiple zones'](https://kubernetes.io/docs/setup/best-practices/multiple-zones/#nodes-are-labeled).
+
+You can check your nodes are labeled via:
+
+```sh
+kubectl get nodes -o custom-columns="name:metadata.name","rack\\zone:metadata.labels.failure-domain\.beta\.kubernetes\.io/zone"
+```
+
+The output might be something like:
+
+```sh
+name                                            rack\zone
+ip-10-0-x-a.eu-central-1.compute.internal    eu-central-1a
+ip-10-0-x-b.eu-central-1.compute.internal    eu-central-1a
+ip-10-0-x-c.eu-central-1.compute.internal    eu-central-1b
+ip-10-0-x-d.eu-central-1.compute.internal    eu-central-1b
+```
+
+All you need to do is tell the operator which label represents the rack zone:
+
+```yaml
+apiVersion: app.redislabs.com/v1
+kind: RedisEnterpriseCluster
+metadata:
+  name: example-redisenterprisecluster
+spec:
+  size: 3
+  rackAwarenessNodeLabel: failure-domain.beta.kubernetes.io/zone
+```
+
+The operator will change the `topologyKey` for the pod anti-affinity rule to
+the label value used. Combining `rackAwarenessNodeLabel` and `podAntiAffinity`
+requires care to ensure you make the same change.
```

Comment:
Need to add a section around tolerations which will be introduced in 6.0. 
The primary use-case being running Redis Enterprise on a dedicated set of nodes.
https://redislabs.atlassian.net/browse/RED-33069

Reply:
I have added a section on the use of the `podTolerations` property.